[Discover] Improve empty state page

andreadelrio opened the pull request:
## Summary

Closes #86765

- Improves the "no results" page in Discover 
- Added illustration provided by the visual design team
- Added the option to remove existing filters as a way to help the user get results.

![image](https://user-images.githubusercontent.com/4016496/126374657-78c55911-814d-495b-93b2-64dec5ac5d93.png)

### Checklist

Delete any items that are not applicable to this PR.

- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] Any UI touched in this PR is usable by keyboard only (learn more about [keyboard accessibility](https://webaim.org/techniques/keyboard/))
- [ ] Any UI touched in this PR does not create any new axe failures (run axe in browser: [FF](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/), [Chrome](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US))
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the [cloud](https://github.com/elastic/cloud) and added to the [docker list](https://github.com/elastic/kibana/blob/c29adfef29e921cc447d2a5ed06ac2047ceab552/src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker)
- [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server))
- [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)


### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/)
* Commit: 785894efe7b0e1643f4cb515317e2db2855dc38f
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/785894efe7b0e1643f4cb515317e2db2855dc38f)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/d655c05ef0bde2f47e0568555c6edc354b8d06e0) | [after](https://github.com/elastic/kibana/commit/785894efe7b0e1643f4cb515317e2db2855dc38f) | diff |
| --- | --- | --- | --- |
| `discover` | 366 | 376 | +10 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/d655c05ef0bde2f47e0568555c6edc354b8d06e0) | [after](https://github.com/elastic/kibana/commit/785894efe7b0e1643f4cb515317e2db2855dc38f) | diff |
| --- | --- | --- | --- |
| `discover` | 428.4KB | 491.4KB | +63.0KB |

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/","number":"134914","commit":"785894efe7b0e1643f4cb515317e2db2855dc38f"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/)
* Commit: 9c5ff7bb1ca10afc1c425ac0e481f0008e25e432
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/9c5ff7bb1ca10afc1c425ac0e481f0008e25e432)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/d655c05ef0bde2f47e0568555c6edc354b8d06e0) | [after](https://github.com/elastic/kibana/commit/9c5ff7bb1ca10afc1c425ac0e481f0008e25e432) | diff |
| --- | --- | --- | --- |
| `discover` | 366 | 376 | +10 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/d655c05ef0bde2f47e0568555c6edc354b8d06e0) | [after](https://github.com/elastic/kibana/commit/9c5ff7bb1ca10afc1c425ac0e481f0008e25e432) | diff |
| --- | --- | --- | --- |
| `discover` | 428.4KB | 489.8KB | +61.4KB |

### History
* :green_heart: [Build #134914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/) succeeded 785894efe7b0e1643f4cb515317e2db2855dc38f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/","number":"134914","commit":"785894efe7b0e1643f4cb515317e2db2855dc38f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/","number":"136373","commit":"9c5ff7bb1ca10afc1c425ac0e481f0008e25e432"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/kibana-app (Team:KibanaApp)

andreadelrio commented:
<img width="1440" alt="Frame 1" src="https://user-images.githubusercontent.com/4016496/125371504-bedf2900-e335-11eb-887d-c3ea7ab22fec.png">

@gchaps can we please get your input for the copy on the pink rectangle?

gchaps commented:
### Short version

**No results match your search criteria**

- If you are searching time series data, try searching over a larger period of time.
- If you applied filters, try [disabling them](https://www.elastic.co/) to reveal more matches.
- If  you still don't see any results, try searching for a different combination of terms.

### Using the existing format

**No results match your search criteria**

**Expand the time range**
You are searching time series data and no matches were found in the specified time range. Try searching over a larger period of time.

**Remove filters from your search**
You have applied filters to narrow the search results. Try removing filters to reveal more matches.

Temporarily remove all filters

**Modify your query**
If you still don’t see any results, there might just not be any matches for your query.  Try searching for a different combination of terms.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/)
* Commit: bd340df537bf5bb1a5f229995903cafd4b0a555e
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check i18n](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/execution/node/430/log)

### History
* :green_heart: [Build #136373](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/) succeeded 9c5ff7bb1ca10afc1c425ac0e481f0008e25e432
* :green_heart: [Build #134914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/) succeeded 785894efe7b0e1643f4cb515317e2db2855dc38f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/","number":"134914","commit":"785894efe7b0e1643f4cb515317e2db2855dc38f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/","number":"136373","commit":"9c5ff7bb1ca10afc1c425ac0e481f0008e25e432"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/","number":"138304","commit":"bd340df537bf5bb1a5f229995903cafd4b0a555e"}]}
PIPELINE-->

andreadelrio commented:
Thank you @ryankeairns for pointing that out and to @gchaps for the suggestions. I've simplified the copy as shown below to cover for three scenarios a) a query exists and there are no filters b) a query exists and there's filters c) there are filters and there's no query. Let me know what you think.

###  **a) A query exists and there are no filters**
![onlyQuery](https://user-images.githubusercontent.com/4016496/125529682-16cb5f8c-ccb6-4fd0-b39d-10d2c38c38f8.png)
### **b) A query exists and there's filters**
![query+filters](https://user-images.githubusercontent.com/4016496/125529679-b086251a-40e2-46c5-b924-d6509869c6e8.png)
### **c) There are filters and there's no query**
![onlyFilters](https://user-images.githubusercontent.com/4016496/125529676-6ed10e25-f215-4cc0-b1b2-00adb23790b7.png)

@timroes I added a big _if_ statement that I'm guessing needs a cleanup, would appreciate your support on that side.

gchaps commented:
Looks good, but it feels to me that the two paragraphs should either have headings or use bullets to show that they are separate ideas.

timroes commented:
We changed the wording to "remove filters". Since there is a difference between removing them (which makes them gone) and just disabling them, I think we should continue calling it "disable filters" instead.

@andreadelrio I'll have a look at the code/if statement, once we settled on the wording and criterias applied to it :-)

ryankeairns commented:
+1 to @gchaps point. 

@andreadelrio perhaps we could use the `EuiDescriptionList` component to achieve the section headings. This will help with scannability and may encourage users to read the longer text if the short title seems relevant to them.

Visually, it would good to see how this looks with a more even amount of padding around all four sides. The heavy title may have extra padding so, practically speaking, this may require setting some slightly different padding values to achieve the more-even visual effect.

Let me know if that is not clear, I feel I got a bit wordy :)

andreadelrio commented:
> * It feels to me that the two paragraphs should either have headings or use bullets to show that they are separate ideas.
> * For this text:
> 
> > You have narrowed your search results.  Try searching for a different combination of terms or removing filters to remove more matches.
> 
> "narrowing search results" is specific to filtering, so I don't think we should include the text about a different combination of terms. Can you use:
> 
> > You have applied filters to narrow the search results. Try removing filters to reveal more matches.
> 
> * Would you ever have a case where there are filters and no query?  The title indicates that there was a query.

Considering this and Tim's comment above, here are the updated three scenarios. @gchaps can you have another look please?

### **1. Only filters**

Heading: Adjust your search
Description: You have narrowed your search results. Try temporarily disabling filters to reveal more matches.

### **2. Filters and query**

Heading: Adjust your search
Description: You have entered a query and have narrowed your search results. Try searching for a different combination of terms or temporarily disabling filters to reveal more matches.

### **3. Only query**

Heading: Adjust your search
Description: Try searching for a different combination of terms to reveal more matches.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/)
* Commit: 57032cb9b168c9f1cf2ccbe9d2d607ce51001aac
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/57032cb9b168c9f1cf2ccbe9d2d607ce51001aac)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/d655c05ef0bde2f47e0568555c6edc354b8d06e0) | [after](https://github.com/elastic/kibana/commit/57032cb9b168c9f1cf2ccbe9d2d607ce51001aac) | diff |
| --- | --- | --- | --- |
| `discover` | 366 | 376 | +10 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/d655c05ef0bde2f47e0568555c6edc354b8d06e0) | [after](https://github.com/elastic/kibana/commit/57032cb9b168c9f1cf2ccbe9d2d607ce51001aac) | diff |
| --- | --- | --- | --- |
| `discover` | 428.4KB | 490.0KB | +61.6KB |

### History
* :broken_heart: [Build #138304](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/) failed bd340df537bf5bb1a5f229995903cafd4b0a555e
* :green_heart: [Build #136373](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/) succeeded 9c5ff7bb1ca10afc1c425ac0e481f0008e25e432
* :green_heart: [Build #134914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/) succeeded 785894efe7b0e1643f4cb515317e2db2855dc38f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/","number":"134914","commit":"785894efe7b0e1643f4cb515317e2db2855dc38f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/","number":"136373","commit":"9c5ff7bb1ca10afc1c425ac0e481f0008e25e432"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/","number":"138304","commit":"bd340df537bf5bb1a5f229995903cafd4b0a555e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/","number":"138558","commit":"57032cb9b168c9f1cf2ccbe9d2d607ce51001aac"}]}
PIPELINE-->

ryankeairns commented:
Perhaps we can consolidate this into two discrete items and not handle the combined version.

### Adjust your filters
Try removing or temporarily disabling filters.

### Adjust your query
Try searching for a different combination of terms.

cc:/ @gchaps

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/)
* Commit: 6f840772e25e986b325f7d4072d973c2937c2b34
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/6f840772e25e986b325f7d4072d973c2937c2b34)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroupDocker](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/execution/node/761/log)
* [Execute xpack-kibana-ciGroupDocker](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/execution/node/808/log)

### History
* :green_heart: [Build #138558](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/) succeeded 57032cb9b168c9f1cf2ccbe9d2d607ce51001aac
* :broken_heart: [Build #138304](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/) failed bd340df537bf5bb1a5f229995903cafd4b0a555e
* :green_heart: [Build #136373](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/) succeeded 9c5ff7bb1ca10afc1c425ac0e481f0008e25e432
* :green_heart: [Build #134914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/) succeeded 785894efe7b0e1643f4cb515317e2db2855dc38f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/","number":"134914","commit":"785894efe7b0e1643f4cb515317e2db2855dc38f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/","number":"136373","commit":"9c5ff7bb1ca10afc1c425ac0e481f0008e25e432"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/","number":"138304","commit":"bd340df537bf5bb1a5f229995903cafd4b0a555e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/","number":"138558","commit":"57032cb9b168c9f1cf2ccbe9d2d607ce51001aac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/","number":"139597","commit":"6f840772e25e986b325f7d4072d973c2937c2b34"}]}
PIPELINE-->

andreadelrio commented:
> Perhaps we can consolidate this into two discrete items and not handle the combined version.
> 
> #### Adjust your filters
> Try removing or temporarily disabling filters.
> 
> #### Adjust your query
> Try searching for a different combination of terms.
> 
> cc:/ @gchaps

I like this approach! I updated the code to have these discrete items. This should be ready for a design review @ryankeairns

timroes commented:
@elasticmachine merge upstream

timroes reviewed (approved):
Code and texts LGTM

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/)
* Commit: f92412f4081148c2ef87eb5703615725d30dffc9
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/f92412f4081148c2ef87eb5703615725d30dffc9)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/85f726c3d88062b7a1a3d724af227943bb5ee670) | [after](https://github.com/elastic/kibana/commit/f92412f4081148c2ef87eb5703615725d30dffc9) | diff |
| --- | --- | --- | --- |
| `discover` | 378 | 388 | +10 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/85f726c3d88062b7a1a3d724af227943bb5ee670) | [after](https://github.com/elastic/kibana/commit/f92412f4081148c2ef87eb5703615725d30dffc9) | diff |
| --- | --- | --- | --- |
| `discover` | 425.1KB | 486.5KB | +61.3KB |

### History
* :broken_heart: [Build #139597](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/) failed 6f840772e25e986b325f7d4072d973c2937c2b34
* :green_heart: [Build #138558](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/) succeeded 57032cb9b168c9f1cf2ccbe9d2d607ce51001aac
* :broken_heart: [Build #138304](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/) failed bd340df537bf5bb1a5f229995903cafd4b0a555e
* :green_heart: [Build #136373](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/) succeeded 9c5ff7bb1ca10afc1c425ac0e481f0008e25e432
* :green_heart: [Build #134914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/) succeeded 785894efe7b0e1643f4cb515317e2db2855dc38f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134914/","number":"134914","commit":"785894efe7b0e1643f4cb515317e2db2855dc38f"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/","number":"136373","commit":"9c5ff7bb1ca10afc1c425ac0e481f0008e25e432"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/","number":"138304","commit":"bd340df537bf5bb1a5f229995903cafd4b0a555e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/","number":"138558","commit":"57032cb9b168c9f1cf2ccbe9d2d607ce51001aac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/","number":"139597","commit":"6f840772e25e986b325f7d4072d973c2937c2b34"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/","number":"139707","commit":"f92412f4081148c2ef87eb5703615725d30dffc9"}]}
PIPELINE-->

gchaps commented:
@andreadelrio Can you provide an updated screenshot?

ryankeairns commented:
Thanks for the copy updates.

From a visual design perspective, let's try and match this up more closely to the 'No index patterns' empty state. For example: restricted/max-width, smaller illustration (?), double check typography, padding, and margins, etc.

<img width="1770" alt="Screen Shot 2021-07-20 at 10 31 25 AM" src="https://user-images.githubusercontent.com/446285/126352591-cd44f4dd-8b7f-4408-8db2-a4b3a015b0bc.png">

andreadelrio commented:
> @andreadelrio Can you provide an updated screenshot?

@gchaps here you go. Do you think we should reduce the first item? it is now a bit lengthy compared to the other ones.
![image](https://user-images.githubusercontent.com/4016496/126246104-1f3641a2-d635-4870-bdc5-6be4167db9f0.png)

gchaps commented:
Yes. Remove the sentence "You are searching time series data..."  Sarting with "Try searching....", makes it parallel with the other two.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139838/)
* Commit: 7313978f3059c77e761f392c9c223c299af31608
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/7313978f3059c77e761f392c9c223c299af31608)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/85f726c3d88062b7a1a3d724af227943bb5ee670) | [after](https://github.com/elastic/kibana/commit/7313978f3059c77e761f392c9c223c299af31608) | diff |
| --- | --- | --- | --- |
| `discover` | 378 | 388 | +10 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/85f726c3d88062b7a1a3d724af227943bb5ee670) | [after](https://github.com/elastic/kibana/commit/7313978f3059c77e761f392c9c223c299af31608) | diff |
| --- | --- | --- | --- |
| `discover` | 425.1KB | 492.8KB | +67.6KB |

### History
* :green_heart: [Build #139707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/) succeeded f92412f4081148c2ef87eb5703615725d30dffc9
* :broken_heart: [Build #139597](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/) failed 6f840772e25e986b325f7d4072d973c2937c2b34
* :green_heart: [Build #138558](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/) succeeded 57032cb9b168c9f1cf2ccbe9d2d607ce51001aac
* :broken_heart: [Build #138304](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/) failed bd340df537bf5bb1a5f229995903cafd4b0a555e
* :green_heart: [Build #136373](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/) succeeded 9c5ff7bb1ca10afc1c425ac0e481f0008e25e432

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/136373/","number":"136373","commit":"9c5ff7bb1ca10afc1c425ac0e481f0008e25e432"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/","number":"138304","commit":"bd340df537bf5bb1a5f229995903cafd4b0a555e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/","number":"138558","commit":"57032cb9b168c9f1cf2ccbe9d2d607ce51001aac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/","number":"139597","commit":"6f840772e25e986b325f7d4072d973c2937c2b34"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/","number":"139707","commit":"f92412f4081148c2ef87eb5703615725d30dffc9"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139838/","number":"139838","commit":"7313978f3059c77e761f392c9c223c299af31608"}]}
PIPELINE-->

gchaps commented:
just noticed this.  "larger" should be "longer".

Try searching over a longer period of time.

ryankeairns reviewed (approved):
Thanks for the adjustments. The copy and result (in the description) look great!

gchaps reviewed (approved):
Copy LGTM

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140166/)
* Commit: 69c10d1030420cfb4ee04f37266f07f00cf4cc1c
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/69c10d1030420cfb4ee04f37266f07f00cf4cc1c)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute kibana-ciGroup12](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140166/execution/node/933/log)
* [Execute kibana-ciGroup12](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140166/execution/node/1092/log)

### History
* :green_heart: [Build #139838](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139838/) succeeded 7313978f3059c77e761f392c9c223c299af31608
* :green_heart: [Build #139707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/) succeeded f92412f4081148c2ef87eb5703615725d30dffc9
* :broken_heart: [Build #139597](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/) failed 6f840772e25e986b325f7d4072d973c2937c2b34
* :green_heart: [Build #138558](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/) succeeded 57032cb9b168c9f1cf2ccbe9d2d607ce51001aac
* :broken_heart: [Build #138304](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/) failed bd340df537bf5bb1a5f229995903cafd4b0a555e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138304/","number":"138304","commit":"bd340df537bf5bb1a5f229995903cafd4b0a555e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/","number":"138558","commit":"57032cb9b168c9f1cf2ccbe9d2d607ce51001aac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/","number":"139597","commit":"6f840772e25e986b325f7d4072d973c2937c2b34"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/","number":"139707","commit":"f92412f4081148c2ef87eb5703615725d30dffc9"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139838/","number":"139838","commit":"7313978f3059c77e761f392c9c223c299af31608"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140166/","number":"140166","commit":"69c10d1030420cfb4ee04f37266f07f00cf4cc1c"}]}
PIPELINE-->

andreadelrio commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140205/)
* Commit: dc1d0c2db1c263d8ff113967580463c496673170
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103602/dc1d0c2db1c263d8ff113967580463c496673170)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/3f4032831dfa3945cb0be0be9469dc13f99d582b) | [after](https://github.com/elastic/kibana/commit/dc1d0c2db1c263d8ff113967580463c496673170) | diff |
| --- | --- | --- | --- |
| `discover` | 378 | 388 | +10 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/3f4032831dfa3945cb0be0be9469dc13f99d582b) | [after](https://github.com/elastic/kibana/commit/dc1d0c2db1c263d8ff113967580463c496673170) | diff |
| --- | --- | --- | --- |
| `discover` | 425.1KB | 492.8KB | +67.6KB |

### History
* :broken_heart: [Build #140166](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140166/) failed 69c10d1030420cfb4ee04f37266f07f00cf4cc1c
* :green_heart: [Build #139838](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139838/) succeeded 7313978f3059c77e761f392c9c223c299af31608
* :green_heart: [Build #139707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/) succeeded f92412f4081148c2ef87eb5703615725d30dffc9
* :broken_heart: [Build #139597](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/) failed 6f840772e25e986b325f7d4072d973c2937c2b34
* :green_heart: [Build #138558](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/) succeeded 57032cb9b168c9f1cf2ccbe9d2d607ce51001aac

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/138558/","number":"138558","commit":"57032cb9b168c9f1cf2ccbe9d2d607ce51001aac"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139597/","number":"139597","commit":"6f840772e25e986b325f7d4072d973c2937c2b34"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139707/","number":"139707","commit":"f92412f4081148c2ef87eb5703615725d30dffc9"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/139838/","number":"139838","commit":"7313978f3059c77e761f392c9c223c299af31608"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140166/","number":"140166","commit":"69c10d1030420cfb4ee04f37266f07f00cf4cc1c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/140205/","number":"140205","commit":"dc1d0c2db1c263d8ff113967580463c496673170"}]}
PIPELINE-->
